### DISCUSSION TITLE
npx-create-payload-app fails - creates empty app folder

### DISCUSSION THREAD

**Discussion by cade-bryant-swbc** [payloadcms/payload#3244] [Q&A] [answered]
To avoid repetition, I had previously posted this problem on StackOverflow., so I will just include the link:

https://stackoverflow.com/questions/77003049/payload-cms-npx-create-payload-app-fails-creates-empty-app-folder-without-fi

Please read it there and tell me what I am doing wrong.

**Comment by denolfe** [comment, marked as answer]
Worked on this in Discord. It appears to be a permissions issue on the development machine.
